docs/data-sources/as_activity_logs.md

```diff
@@ -0,0 +1,73 @@
+---
+subcategory: "Auto Scaling"
+---
+
+# huaweicloud_as_activity_logs
+
+Use this data source to get a list of AS scaling activity logs within HuaweiCloud.
+
+```hcl
+variable "group_id" {}
+
+data "huaweicloud_as_activity_logs" "test" {
+  scaling_group_id = var.group_id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to query the AS scaling activity logs.
+  If omitted, the provider-level region will be used.
+
+* `scaling_group_id` - (Required, String) Specifies the AS scaling group ID.
+
+* `start_time` - (Optional, String) Specifies the start time of the AS scaling activity for query.  
```

Comment:
should add the description of the `start_time` format or an example about this value.

Reply:
fixed.